src/pybind/cephfs/cephfs.pyx

```diff
@@ -150,7 +148,10 @@ cdef extern from "cephfs/libcephfs.h" nogil:
 
 
 class Error(Exception):
-    pass
+    def __init__(self, errno, strerror):
+        super().__init__(errno, strerror)
```

Comment:
`super()` works for new style class. and i think `Exception` is. i tested a minimal reproducer using python2.7 and it works.